cyw43/src/runner.rs

```diff
@@ -73,23 +73,41 @@ where
         }
     }
 
-    pub(crate) async fn init(&mut self, firmware: &[u8]) {
+    pub(crate) async fn init(&mut self, firmware: &[u8], bluetooth_firmware_offsets: Option<&[(u32, usize)]>, bluetooth_firmware: Option<&[u8]>) {
```

Comment:
this is a "breaking change", let me know if you want me to make a different `init_with_bluetooth`` function  or something or update all of the other examples

Reply:
Perhaps `struct Firmware { wifi: &[u8], bluetooth: Option<&[u8]> }`? 

The format for the bluetooth binary is described here -> https://github.com/georgerobotics/cyw43-driver/blob/main/firmware/README.md

It'd be simpler for the user if they could pass a `&[u8]` with that format, it's 1 thing instead of 2. Also it'd make updating the firmware easier in the future.

Reply:
updated, let me know what you think